pkg/operator/sync.go

```diff
@@ -55,9 +55,11 @@ func (optr *Operator) syncClusterAPIController(config OperatorConfig) error {
 	controller := resourceread.ReadDeploymentV1OrDie(controllerBytes)
 	_, updated, err := resourceapply.ApplyDeployment(optr.kubeClient.AppsV1(), controller)
 	if err != nil {
+		glog.Error("Failed to apply deployment")
```

Comment:
You might as well log the underlying error here too.
However, do we know why the return is not logged further up the call chain?

Reply:
The underlying error is logged after this function returns.  Unfortunately, the class of error can be 'timeout' and that's not specific to any particular thing this function does, so there's no way to see what 'actually' failed in this function.

This is basically just a bread crumb, I don't actually care what the error was just that I couldn't apply the deployment.